pkg/features/features.go

```diff
@@ -50,6 +50,12 @@ const (
 	// MountChangeDetection feature flag is used to enable/disable
 	// mount point and fs type change detection for block devices
 	MountChangeDetection = "MountChangeDetection"
+
+	// PartitionTableUUID feature flag is used to enable use a
+	// partition table uuid instead of create partition described in
+	// https://github.com/openebs/node-disk-manager/issues/621 .
+	// This feature must enabled with GPTBasedUUID.
+	PartitionTableUUID Feature = "PartitionTableUUID"
```

Comment:
Can you add one check, to make sure this feature gets enabled only with GPTBasedUUID. The check can be added [here](https://github.com/openebs/node-disk-manager/pull/635/files#diff-4914632202b9c7ee5d0ba5f8d143ba5a91887e66c02f35e1456e912092a9346dR134), so that such dependent features can be checked in future also.

Reply:
I think you missed this comment. PTAL.

Reply:
@akhilerm Sorry, the `here` link previously point to the `pkg/partition/partition.go` file, so i add the feature check in that files. What the `here` point to exactly ?

Reply:
oops. sorry for that, the check should be in the feature pkg [here](https://github.com/openebs/node-disk-manager/blob/develop/pkg/features/features.go#L126). So everything related to features are handled there.

Reply:
Also, the check needed is, if PartitionTableUUID feature gate is given , we should make sure that GPTBasedUUID is also provided. The best place to make that check is in the feature pkg.

Reply:
OK, I get that. I will fix it!